dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php

```diff
@@ -201,11 +201,16 @@ private function getFullWhitelist()
 
     public function testCodeStyle()
     {
+        $useWhiteList = isset($_ENV['phpunit.static.testcodestyle.use_whitelist'])
+            && $_ENV['phpunit.static.testcodestyle.use_whitelist'] === '1';
```

Comment:
Simply `$whitelist = !empty(...is_full_scan...) ? ... `

Please use both `php` and `phtml`.